Automattic/newspack-custom-content-migrator
# Move MigrationMeta to NMT

## How to test
This moves the MigrationMeta logic to the NMT. See https://github.com/Automattic/newspack-migration-tools/pull/8 

---
